src/search/landmarks/landmark_heuristic.cc

```diff
@@ -74,8 +76,7 @@ void LandmarkHeuristic::compute_landmark_graph(const plugins::Options &opts) {
         log << "Generating landmark graph..." << endl;
     }
 
-    shared_ptr<LandmarkFactory> lm_graph_factory =
-        opts.get<shared_ptr<LandmarkFactory>>("lm_factory");
+    auto lm_graph_factory = opts.get<shared_ptr<LandmarkFactory>>("lm_factory");
```

Comment:
My IDE or rather clang-tidy seems to recommend to "use `auto` when initializing with a timplate cast to avoid duplicating the type name". I don't have a strong opinion myself, so I'm fine with changing it back.